packages/oauth-twitter/src/index.ts

```diff
@@ -0,0 +1,52 @@
+import * as oauth from 'oauth';
+
+export interface AccountsOauthTwitterOptions {
+  key: string;
+  secret: string;
+}
+
+class AccountsOauthTwitter {
```

Comment:
This is a really minor comment, feel free to disregard.

For consistency, let's always use `OAuth` instead of `Oauth`. So instead of `AccountsOauthTwitterOptions` it'll be `AccountsOAuthTwitterOptions`. The reason for this is because the `oauth` library has a constructor `OAuth`.